src/pykiso/logging_initializer.py

```diff
@@ -234,3 +239,94 @@ def initialize_loggers(loggers: Optional[List[str]]) -> None:
     loggers_to_deactivate = set(relevant_loggers) - set(active_loggers)
     for logger_name in loggers_to_deactivate:
         logging.getLogger(logger_name).setLevel(logging.WARNING)
+
+
+def import_object(path: str) -> Any:
+    """return the object based on the path.
+        For example : logging.Logger will return Logger
+
+    :param path: path to the object
```

Comment:
dont we need :return: for sphinx ? i know that your current description describes the return value

Reply:
Done